books.js

```diff
@@ -0,0 +1,21 @@
+const Books = [{
```

Comment:
This is cool for now, but I still think we should use a real database, if we have time, after we get everything else in order.

Reply:
Actually, db is not really necessary I guess.